treeherder/webapp/api/bugzilla.py

```diff
@@ -34,7 +34,10 @@ def create_bug(self, request):
         ).encode("utf-8")
         summary = params.get("summary").encode("utf-8").strip()
         url = settings.BUGFILER_API_URL + "/rest/bug"
-        headers = {"x-bugzilla-api-key": settings.BUGFILER_API_KEY, "Accept": "application/json"}
+        headers = {
+            "x-bugzilla-api-key": settings.BUGFILER_API_KEY,
```

Comment:
FYI:
I set the buzilla api url[0] and api key[1] for the test through hardcoding on my local.

[0] BUGFILER_API_URL: "https://bugzilla.mozilla.org"
[1] BUGFILER_API_KEY: "My bugzilla api key"